constellation-app/constellation
# Adding Undo Redo Auto notes in notes view

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change
Auto Notes are now generated for Undo Redo actions on the graph which can be filtered with the tag `Undo/Redo`
Actions which do not have notes added such as zoom, drag would not add the respective undo redo notes.



### Alternate Designs



### Why Should This Be In Core?



### Benefits



### Possible Drawbacks



### Verification Process

1. Open a graph and open Notes View and have the auto notes filter on
2. Add a node and observe the note generated in notes view
3. Undo that add (Ctrl-Z) and observe the note generated
(Redo doesn't work currently due to a bug in Notes view which will be handled separately.)



### Applicable Issues
https://github.com/constellation-app/constellation/issues/1141
